db/migrate/20221122150342_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :comments_counter
+      t.integer :likes_counter
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] The references should point to an `author`, not `user`. You can use the `to_table` to point the database to the User table.

  You can refer to this thread on naming references to point to a different table: [https://stackoverflow.com/questions/13694654/specifying-column-name-in-a-references-migration](https://stackoverflow.com/questions/13694654/specifying-column-name-in-a-references-migration)

Reply:
@DammyShittu thank you very much for your review and the help. Happy codinng! 🚀